packages/dashboard/src/pages/Personas/hooks/usePersonaProof.ts

```diff
@@ -0,0 +1,14 @@
+import { queryExistedBindingByPersona } from '@masknet/web3-providers'
+import { useEffect } from 'react'
+import { useAsyncRetry } from 'react-use'
+import { Messages } from '../../../API'
+import { PersonaContext } from './usePersonaContext'
+
+export function usePersonaProof() {
+    const { currentPersona } = PersonaContext.useContainer()
+    const res = useAsyncRetry(async () => {
+        return queryExistedBindingByPersona(currentPersona?.publicHexKey as string)
+    }, [])
```

Comment:
Should depend currentPersona?.publicKey?
And please remove `as string`.

Reply:
as string removd,

Reply:
refacted